pnpm-lock.yaml

```diff
@@ -14770,6 +14774,11 @@ packages:
       unist-util-visit: 3.1.0
     dev: true
 
+  /mdast/3.0.0:
+    resolution: {integrity: sha512-xySmf8g4fPKMeC07jXGz971EkLbWAJ83s4US2Tj9lEdnZ142UP5grN73H1Xd3HzrdbU5o9GYYP/y8F9ZSwLE9g==}
+    deprecated: '`mdast` was renamed to `remark`'
```

Comment:
🤔 Hmm, can you get `Image` from `remark` or is it only available in the deprecated package?

Reply:
Ah, [looks like the remark docs recommend `@types/mdast` without installing the core dep.](https://github.com/remarkjs/remark#types) Only `mdast` is deprecated while the types are stable, so I uninstalled that.